app/Components/SudokuBoard/Components/EndGameModal.tsx

```diff
@@ -20,9 +28,28 @@ const EndGameModal = (props: EndGameModalProps) => {
   const theme = useTheme();
   const navigation: any = useNavigation();
 
+  // sort by most number of hints
```

Comment:
line 31 through 48 is duplicated in TotalStatistics.tsx, is it possible to move it all into a function that returns strategyHints?

Reply:
yeah we can, I was just lazy. I'll do that

Reply:
I made a shared component for this to get rid of the duplication, good call